### PR TITLE
Add localhost:3000 into excluded urls

### PR DESCRIPTION
**Pull request by well-balanced**
## Changes
Related issue: #3800 

- Fix the `create_team_with_test_account_filters` test code.
- Add host `localhost:3000` into initial value.
## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

**Comment by well-balanced** [comment]
I'm so glad I could contribute even a little bit. 

if there are something to fix, please tell me about that. thx :)

**Comment by yakkomajuri** [comment]
Thanks a lot for this @well-balanced !

**Comment by yakkomajuri** [comment]
I don't have a lot of context here but it seems like we had a migration for this before, to add this to existing teams:

https://github.com/PostHog/posthog/blob/bf2c4429b51b780dc957a9908f1ddced0429b911/posthog/migrations/0132_team_test_account_filters.py

**Comment by well-balanced** [comment]
Oh I see that !
I'll update it :)

File: posthog/migrations/0132_team_test_account_filters.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def forward(apps, schema_editor):
             {
                 "key": "$host",
                 "operator": "is_not",
-                "value": ["localhost:8000", "localhost:5000", "127.0.0.1:8000", "127.0.0.1:3000"],
+                "value": ["localhost:8000", "localhost:5000", "127.0.0.1:8000", "127.0.0.1:3000", "localhost:3000"],
```
Comment (well-balanced) [inline posthog/migrations/0132_team_test_account_filters.py:43]: @yakkomajuri  Could you check it one more time?

**Comment by yakkomajuri** [comment]
Ah sorry @well-balanced, you shouldn't update an old migration. Rather, I was wondering if we should have a new one. 

However, thinking again, I think this is small enough of a change that we probably don't need to add it to old teams, we can maybe live with just new teams having this. 

Could you revert that last commit please? And sorry for the confusion here.

**Comment by well-balanced** [comment]
> Ah sorry @well-balanced, you shouldn't update an old migration. Rather, I was wondering if we should have a new one.
> 
> However, thinking again, I think this is small enough of a change that we probably don't need to add it to old teams, we can maybe live with just new teams having this.
> 
> Could you revert that last commit please? And sorry for the confusion here.

@yakkomajuri  I forgot the migration was sequential. I reverted a troubled commit !

**Comment by timgl** [comment]
Thanks for doing this!

**Pull request merged**